src/routes/(console)/project-[project]/settings/smtp/+page.svelte

```diff
@@ -65,6 +65,13 @@
                 username = undefined;
                 password = undefined;
             }
+
+            let secureProtocol = undefined;
```

Comment:
do we need to create a new variable? Can we just pass in `secure`?

Reply:
Fixed